### ISSUE TITLE
Training Results Are Not Reproducible

### ISSUE THREAD

**Issue by EuroMinyoung186** [xuanyuzhang21/EditGuard#14]
I am encountering an issue where the training process does not produce reproducible results, even when the random seed is set. Additionally, during Stage 1, where both BEM and BRM are trained, the message MSE loss does not decrease as expected. Instead, it increases and often reaches around 0.5. What is problem in my training set?

**Comment by xuanyuzhang21** [comment]
This is strange. I just re-ran the training code of stage 1, and it seems that the loss can be reduced normally, and the bit error rate also dropped rapidly to about 1%. The following is my training log. Due to time constraints, I only trained about 8,000 iters. In fact, it takes longer to train to achieve better results. I guess it may be because the coco dataset I used eliminated images with a length or width less than 400, in order to filter out more high-quality skilled data. This may help you reproduce it. Thanks~
[train_train_ibsn_bit_64_250109-115737.log](https://github.com/user-attachments/files/18361727/train_train_ibsn_bit_64_250109-115737.log)

**Comment by LIUTIGHE** [comment]
Hi may I know how long it took to finish the training?
